projects/lib/src/board/board.h

```diff
@@ -309,11 +311,14 @@ class LIB_EXPORT Board
 		 * \param symbol Short piece name or piece symbol
 		 * \param movement A bit mask for the kinds of moves the
 		 *        piece can make.
+		 * \param gSymbol Select the piece's graphical representation.
+		 *	  If not set the \a symbol will be used (default).
 		 */
 		void setPieceType(int type,
 				  const QString& name,
 				  const QString& symbol,
-				  unsigned movement = 0);
+				  unsigned movement = 0,
+				  const QString & gSymbol = 0);
```

Comment:
The default value of 0 is a bit weird when the type is a QString reference instead of a pointer. Am I correct to assume that the zero value invokes the `QString::QString(const char *str)` constructor? If that's the case, I'd rather use a default value of `QString()` to contruct a null string.